GitCommands/Git/GitModule.cs

```diff
@@ -2041,22 +1984,6 @@ public bool InTheMiddleOfAction()
             return InTheMiddleOfConflictedMerge() || InTheMiddleOfRebase();
         }
 
-        public ArgumentString CommitCmd(bool amend, bool signOff = false, string author = "", bool useExplicitCommitMessage = true, bool noVerify = false, bool gpgSign = false, string gpgKeyId = "", bool allowEmpty = false, bool resetAuthor = false)
-        {
-            return new GitArgumentBuilder("commit")
-            {
-                { amend, "--amend" },
-                { noVerify, "--no-verify" },
-                { signOff, "--signoff" },
-                { !string.IsNullOrEmpty(author), $"--author=\"{author?.Trim().Trim('"')}\"" },
-                { gpgSign && string.IsNullOrWhiteSpace(gpgKeyId), "-S" },
-                { gpgSign && !string.IsNullOrWhiteSpace(gpgKeyId), $"-S{gpgKeyId}" },
-                { useExplicitCommitMessage, $"-F \"{GetGitExecPath(Path.Combine(GetGitDirectory(), "COMMITMESSAGE"))}\"" },
```

Comment:
This change prevents commits in WSL.
Only in master.
Not sure if other commands are affected, refactoring moving code are hard to review.

Reply:
This is the only real change. A comment would have been good. How does this make it work in WSL?

Reply:
The path is in the format for gitexec, ie in wsl format.
Added for a few commands, manually identified.

This was a series of changes, I did not see any related changes. So hopefully the only issue. 

The transformation will have to be added to the caller of commands arguments where module is available.